docs-ref-conceptual/azure-cli-vm-tutorial-4.md

```diff
@@ -28,13 +28,26 @@ to get more information about the VM's configuration and capabilities.
 In order to extract the object ID, the `--query` argument is used. Queries are written in the [JMESPath query language](http://jmespath.org)Start
 with getting the network interface controller (NIC) object ID.
 
+# [Bash](#tab/bash)
+
 ```azurecli-interactive
 az vm show --name $vmName \
   --resource-group $resourceGroup \
   --query 'networkProfile.networkInterfaces[].id' \
   --output tsv
 ```
 
+# [PowerShell](#tab/powershell)
```

Comment:
@v-abdriaz , to be consistent with the rest of the article, I added both a Bash and PowerShell conceptual tab here. The only difference is line continuation characters, but we've made this distinction in the other steps, so we need to do the extra work here as well.

This update was completed in your branch so I can merge the PR.